conformance/tests/test_published_numbers_guard.py

```
@@ -56,39 +79,69 @@ def sandbox():
         for manifest in (REPO / "conformance/adequacy").glob("*.manifest.json"):
             shutil.copy2(manifest, adequacy / manifest.name)
 
-        saved = (chk.REPO, chk.ADEQUACY, chk.RESULTS, chk.DOCUMENTS)
+        git = lambda *a: subprocess.run(["git", "-C", str(root), *a],  # noqa: E731
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)